cypress/integration/settings/remote-storage/configure-remote-storage.spec.js

```diff
@@ -0,0 +1,28 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import RemoteStorageHelper from '../../../support/fragments/settings/remote-storage/configuration';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import testTypes from '../../../support/dictionary/testTypes';
+
+describe('export instance records', () => {
+  beforeEach('login', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
```

Comment:
I don't see about minimal permissions. In this text I see that user must have this permission and our diku_admin has it.

Reply:
please check this with the Team. Sometime it can be critical to check special sets of permissions. Additionally you can get failed tests in case when someone change diku_admin permissions.

Reply:
I asked about that PO. When I get answer, I will decide about creating new user

Reply:
I added todo for this, because I still waiting response about that